tests/Functional/Schema/SchemaBuilderTest.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
```

Comment:
@hungneox Could you remove the author block for consistency.

Reply:
@crisu83 I merged it already :< I will remove this on the next PR. 😢